### PR TITLE
fix: deploy to different region bug

### LINKED ISSUE #3020: Deploy to westeurope error
## Describe the bug

after deploy to westeurope, bot have a permission error.
image

## Browser

- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

## OS

- [ ] macOS
- [ ] Windows
- [ ] Ubuntu

### PR DESCRIPTION
**Pull request by VanyLaw**
## Description

Cause: not update the region into appsettings.deployment.json of the release project.
Solusion: update it into appsettings.deployment.json

## Task Item

close #3020

### CODE REVIEW

File: Composer/plugins/azurePublish/src/index.ts [hunk 1]
```diff
@@ -236,8 +225,8 @@ class AzurePublisher {
         publishName,
         environment,
         provision?.MicrosoftAppPassword,
-        luisAuthoringKey,
-        luisAuthoringRegion,
+        provision?.luis?.authoringKey,
```
Comment (benbrown) [inline Composer/plugins/azurePublish/src/index.ts:228]: these are not necessary to include in the GUID generation

File: Composer/plugins/azurePublish/src/index.ts [hunk 2]
```diff
@@ -269,8 +258,8 @@ class AzurePublisher {
         subscriptionID,
         publishName,
         environment,
-        luisAuthoringKey,
-        luisAuthoringRegion,
+        luisAuthoringKey: provision?.luis?.authoringKey,
```
Comment (benbrown) [inline Composer/plugins/azurePublish/src/index.ts:261]: It is not necessary to pull these in - they are already being pulled out of the appsettings.deployment.json file inside the deploy function in `botProjectDeploy`

```
      if (luisSettings) {
        // if luisAuthoringKey is not set, use the one from the luis settings
        luisAuthoringKey = luisAuthoringKey || luisSettings.authoringKey;
        luisAuthoringRegion = luisAuthoringRegion || luisSettings.region;
        luisEndpointKey = luisSettings.endpointKey;
      }
```

**Comment by benbrown** [comment]
Do not merge! This will be superceded by another PR.

**Comment by a-b-r-o-w-n** [comment]
closing for #3029

**Pull request closed without merging**